### DISCUSSION TITLE
I have a 2010 MacBook pro updated to venture 13.3 with open core legacy build and patched having trouble with text completely showing up

### DISCUSSION THREAD

**Discussion by St396** [community/community#51678] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am having trouble on my 2010 MacBook pro non metal mac with text websites showing up completely until for a website on safari I click on another application, the website information shows up. Now for other apps missing information and nothing I can do it does not show up if I simply click on another application. I am not sure if there is a work around or not to this issue I tried looking it up but could not find anything. The pictures show what I am trying to say. I upgraded to venture 13.3 open core legacy patcher 0.6.2. 
![IMG_1294](https://user-images.githubusercontent.com/103712055/229292657-4200a9d4-3b86-486e-bcfd-da8d5de882b7.jpeg)
![IMG_1293](https://user-images.githubusercontent.com/103712055/229292661-cb1225b0-678e-454a-8c97-403c63ce760c.jpeg)

![IMG_1286](https://user-images.githubusercontent.com/103712055/229292554-e3d0de87-b49e-4926-a4b8-1bb47f3133fc.jpeg)

**Comment by queenofcorgis** [comment]
Hi @St396 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Safari or Apple. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
